.travis.yml

```diff
@@ -28,3 +28,4 @@ branches:
 before_install:
   - gem update --system
   - gem install bundler
+  - sudo apt install optipng
```

Comment:
The make target you've added this to is also run on Netlify's container before deployment, where I expect it'll fail due to the lack of the special Travis variable and optipng.  My suggestion is that the Makefile be edited to create a separate `travis` build target that depends on `production` and also includes this test, e.g.:

```makefile
travis: production
  	## Check that newly added or modifyed PNGs are optimized
	_contrib/travis-check-png-optimized.sh
```

Then update the `.travis.yml` to call `make travis` instead of the current `make production`.

Reply:
Oh, it succeeds because there's no diff when that travis variable is unavailable, so maybe this is unnecessary.

Reply:
> Oh, it succeeds because there's no diff when that travis variable is unavailable, so maybe this is unnecessary.

When not using `$TRAVIS_COMMIT_RANGE` this becomes a issue again though. But I'd rather not name the build target `travis`. A developer would need to run `make travis` locally, which could be a bit confusing.

Reply:
Do you have ideas for a different name? I thought maybe `make check`...

Reply:
Maybe you could leave it where it is and make one of the first lines of the script something like `if ! which optipng >/dev/null ; then echo "optipng not installed; skipping test..."; exit ; fi`